index.js

```diff
@@ -104,6 +104,12 @@
       swaggerObject = createSwaggerObject(loopbackApplication, opts);
     });
 
+  // listening to remoteMethodDisabled event for updating the swaggerObject
+  // when a remote method is disabled and remote it from the Swagger UI.
+    loopbackApplication.on('remoteMethodDisabled', function() {
```

Comment:
Hm...its probably the linter that messed up. ;( Could you add a separate commit for the indentation fixes? Much appreciated. ;)